ui/src/plugins/dev.perfetto.Sched/cpu_slice_selection_aggregator.ts

```diff
@@ -81,44 +81,37 @@ export class CpuSliceSelectionAggregator implements Aggregator {
       {
         title: 'Process',
         kind: 'STRING',
-        columnConstructor: Uint16Array,
         columnId: 'process_name',
       },
       {
         title: 'PID',
         kind: 'NUMBER',
-        columnConstructor: Float64Array,
         columnId: 'pid',
       },
       {
         title: 'Thread',
         kind: 'STRING',
-        columnConstructor: Uint16Array,
         columnId: 'thread_name',
       },
       {
         title: 'TID',
         kind: 'NUMBER',
-        columnConstructor: Float64Array,
         columnId: 'tid',
       },
       {
-        title: 'Wall duration (ms)',
-        kind: 'TIMESTAMP_NS',
-        columnConstructor: Float64Array,
+        title: 'Wall duration',
```

Comment:
Why are we removing the units here and not elsewhere? Feels inconsistent.

Reply:
The kind hint 'DURATION_NS' tells the aggregator to expect durations in nanoseconds, and the aggregator will take charge of converting to s/ms/µs/etc according to user preferences.

I've left the other units in because:
- They are using units we have converters or centralized user preferences for - e.g. watts, joules
- They not represented as nanoseconds in the source data - usually floating point milliseconds or similar - and are in some 3rd party plugin I don't particularly want to touch.

Reply:
Ended up rolling back in order to limit the scope of this PR.